### PR TITLE
Add MinIO admin API support.

### PR DESCRIPTION
**Pull request by balamurugana**

### CODE REVIEW

File: minio/minioadmin.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+"""Minio Admin wrapper using Minio Client (mc) tool."""
+
+from __future__ import absolute_import
+
+import json
+import subprocess
+
+
+class MinioAdmin:
+    """Minio Admin wrapper using Minio Client (mc) tool."""
```
Comment (harshavardhana) [inline minio/minioadmin.py:26]:
```suggestion
    """MinIO Admin wrapper using MinIO Client (mc) tool."""
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: minio/minioadmin.py [hunk 2]
```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+# MinIO Python Library for Amazon S3 Compatible Cloud Storage,
+# (C) 2020 MinIO, Inc.
```
Comment (kannappanr) [inline minio/minioadmin.py:3]:
```suggestion
# (C) 2021 MinIO, Inc.
```

**Comment by krisis** [comment]
@balamurugana it would be useful to have a README introducing this package and a couple of simple examples to demonstrate usage like we have for the minio package .

**Review by krisis — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Review by kannappanr — APPROVED** [review]

**Comment by torbendury** [comment]
Is there any documentation available on how to use this `MinioAdmin` object?

**Comment by balamurugana** [comment]
@torbendury If you know how to use `mc admin` command line, you can directly refer https://github.com/minio/minio-py/blob/master/minio/minioadmin.py

**Comment by torbendury** [comment]
@balamurugana That's what I did, but searching through function definitions doesn't feel right. At least a refer on the Python SDK docs to this file would be great, so that other who wonder about the admin API don't wonder how to access it.

**Pull request merged**